### PR TITLE
Feature/add taint and tolerations

### PR DESCRIPTION
**Pull request by cbeneke**

### CODE REVIEW

File: pages/02.Documentation/10.taints-and-tolerations/default.en.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: 'Taints and Tolerations'
+published: true
+taxonomy:
+    tag:
+        - kubernetes
+        - gui
+---
+
+[Taints and Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) are used in Kubernetes to mark single nodes for specific use-cases (taints) and allowing pods to be scheduled on such nodes (tolerations).
+While it is possible to taint single nodes manually we do not recommend to do so when using your MetaKube cluster, since each update of the nodes will recreate them, and they will loose their taints. When creating a Node Deployment we provide a setting, that all nodes get created with Labels and/or Taints.
```
Comment (bashofmann) [inline pages/02.Documentation/10.taints-and-tolerations/default.en.md]: manually over the Kubernetes API with kubectl

File: pages/02.Documentation/10.taints-and-tolerations/default.en.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+![MetaKube Cluster creation Overview](cluster-creation.png)
+
+Newly created machines will then automatically be tainted and/or labelled with the values you chose. In a typical setup one would set a Taint with NoSchedule Effect and a matching label on the node (e.g. Key `dedicated` and Value `web`). Deployments which shall run on these dedicated nodes then need two settings: The toleration to be _allowed_ to be scheduled on these nodes, and a nodeSelector to be _restricted_ to only these nodes:
```
Comment (bashofmann) [inline pages/02.Documentation/10.taints-and-tolerations/default.en.md]: NodeSelector or NodeAffinity

File: pages/02.Documentation/10.taints-and-tolerations/default.en.md (hunk 1 above)
Comment (bashofmann) [reply pages/02.Documentation/10.taints-and-tolerations/default.en.md]: Taints can also be set at the MachineDeployment resource over the CLI. We may want to document that.

Comment (cbeneke) [reply pages/02.Documentation/10.taints-and-tolerations/default.en.md]: Fixed

File: pages/02.Documentation/10.taints-and-tolerations/default.en.md (hunk 2 above)
Comment (cbeneke) [reply pages/02.Documentation/10.taints-and-tolerations/default.en.md]: Fixed

**Pull request merged**